Clima/Controller/WeatherViewController.swift

```diff
@@ -34,7 +46,7 @@ class WeatherViewController: UIViewController, UITextFieldDelegate {
         return true
     }
     
-    func textFieldShouldEndEditing(_ textField: UITextField) -> Bool {
+    internal func textFieldShouldEndEditing(_ textField: UITextField) -> Bool {
```

Comment:
Internal is the default access control and you don't need to used it, please remove the internal access